### DISCUSSION TITLE
G Study Design and Set-up Question

### DISCUSSION THREAD

**Discussion by KatAMarz** [G-String-Legacy/G_String#21] [Q&A]
Hi! **I'm attempting to conduct a g-study and subsequent d-studies for my dissertation.** I have a few questions to ensure the feasibility of the topic. Otherwise, I will have to switch to a completely new topic. Please let me know if you need any other additional information about the study design to help answer questions. Any and all feedback is helpful! I appreciate this community, and thank you for your time!

**Aim**: To calculate the g coefficient and understand sources of variance for an intervention adherence measure of a teacher-delivered behavioral management intervention. The adherence measure is measuring teacher behavior (adherence to an intervention) and is rated by two coders (coder 1 and coder 2) out of a pool of 10 coders. I'm particularly interested in running a d studies to determine how many observations and how many raters are needed for a reliable estimate of adherence.

**Data structure**: The facets, I believe, include teacher, child, observation number, and rater. 
- Each teacher has 2-3 students included in the study. 
- For each teacher-child dyad, adherence is rated at 7 timepoints (observation number). However, not all teachers completed 7 observations, some did more, some did less.
- At each observation, adherence was measured by two coders. However, there were 10 different coders selected randomly to be either coder 1 or coder 2.
- I'm unsure whether item or subscale score should also be considered a facet (and whether it is crossed or nested with what other facets). The adherence measure has 7 items which are averaged for a subscale score.

**G-Study Design:** This design is pretty unbalanced. I think the design would be (o:c:t)r, meaning observations are nested within children which are nested within teacher. These are then crossed with rater, I think. 

**Questions:**
1) Is this study feasible, or should I scrap the idea?
2) Did I define the study correctly with the corrected nested and crossed facets? Do I need to add an item or subscale adherence score as facet (is it crossed or nested and with what)? Are raters crossed, or would they actually be nested? 
3) Would the subject of measurement be teacher, observation, or rater? 
4) Should my excel sheet contain ratings for all 7-items or just the 1 subscale "average" score?
5) Should I handle rater as "rater 1 and rater 2" or unique ids for each of the 10 raters? Or is there another way I should handle this?
6) In g_sttring_L would I "gray out" all columns of data up until the adherence scores? So teacher id, child id, observation number, and rater id are all blocked out?

Thank you so much for your help!

**Comment by Papa-26** [comment]
Hi Kat?
let me start by apologizing that Generalizability Analysis is not a tool to create results from chaos, but only a tool that optimally makes use of existent order within chaos. It works relatively well, when at least one factorial facet of generalization is fixed, applying to all facets of differentiation.
In your case, unfortunately, the only facet of generalization are the raters, whose severity we have to consider as randomly distributed. In the unlikely case that they are absolutely equivalent, your design would have a good chance at working. It is also not clear to me, what the difference is between 'coders', and 'raters'. Wouldn't both have to convert observed behaviours into some form of a scale? It is probably not wise to break this process up. However, in planning your research, it would be cautious to first do a pilot study looking at the consistency of this coding/rating process, by creating, let's us say 8 spontaneous videos of typical situations, e.g. 2 students * 2 teachers x 2 occasions (BTW, I didn't see the students mentioned as facets). Then take, let's say, 5 coders/raters, and evaluate their interrater reliability on these 8 videos. If this is very high, you should be safe.

Best of Luck,
Ralph

**Comment by Papa-26** [comment]
Naomi set me straight: 'c' stands for child, not coder. So the design is: r : o : c : t, and there are no crossed components. So I can't see, how you do a Generalizability Analysis. Sorry to be a bearer of bad news. The problem is that Generalizability Analysis does not keep track of the identity of nested facets.
